scripts/test.sh

```diff
@@ -36,6 +36,8 @@ TEST_PKCS11_MODULE_PATH=${TEST_PKCS11_MODULE_PATH:-/usr/lib/softhsm/libsofthsm2.
 # note: on Ubuntu bionic, use /usr/lib/engines/engine_pkcs11.so on xenial
 TEST_PKCS11_ENGINE_PATH=${TEST_PKCS11_ENGINE_PATH:-/usr/lib/x86_64-linux-gnu/engines-1.1/libpkcs11.so}
 
+BUILD_DOCS=${BUILD_DOCS:-0}
```

Comment:
Would be best if the variable followed the same format as the others (TEST_WITH_DOCS ?) and be grouped with the others , around line 10-14